main.js

```diff
@@ -0,0 +1,138 @@
+  var tableData = postName + postContent + postQuality + postEdit
+  $('#' + i).html(tableData)
+
+  // set the inital quality attribute to zero - it can be changed by clicking thumbs up and thumbs down
+  $('#' + i).attr('qual', 0);
+  //reset the form
+  $('#message').val('')
+  $('#name').val('')
+
+  //add a comment submission form to each new post
+  $('.comment-button').parent().siblings().last().html('<input id=comment' + i + '  type="text" class="form-inline" placeholder="Comment Text"></input><input id=comment-name' + i + ' type="text" class="form-inline" placeholder="Your Name"></input><button type="button" class="btn btn-primary btn-xs	submit-comment">Post Comment</button>')
+
+  //hide the comments and submission form to start out and then toggle them on when "comments" is pressed
+  $('.comment-button').parent().nextAll().hide()
+  var hide = true
+  $('.comment-button').on('click', function() {
+    $(this).parent().nextAll().toggle(hide)
+    hide = !hide
+  })
+
+  $('.submit-comment').on('click', function(event1) {
+    //allows a user to submit a comment and provides an edit and delete icon for the comment
+    event1.stopImmediatePropagation();
+    var commentName = $(this).prev().val();
+    var commentContent = $(this).prev().prev().val()
+    var commentData = "<span class='comments'><span>" + commentContent + "</span><strong> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Posted by:</strong> " + commentName + "&nbsp;&nbsp;&nbsp;<a class='delete-comment'><i class='fa fa-times text-primary' aria-hidden='true'></i></a>&nbsp;<a class='edit-comment'><i class='fa fa-pencil text-primary' aria-hidden='true'></i></a></span>"
+    $(this).parent().prev().append(commentData)
+    $(this).prev().val("");
+    $(this).prev().prev().val("")
+
+    $('.delete-comment').on('click', function(event1) {
+      event1.stopImmediatePropagation();
+      $(this).parent().remove()
+    })
+
+    $('.edit-comment').on('click', function(event1) {
+      event1.stopImmediatePropagation();
+      result = window.prompt("Update your comment", $(this).prev().prev().prev().html());
+      if (result === null) {
+        return;
+      }
+      $(this).prev().prev().prev().html(result)
+    })
+
+  })
+  clickTools(i, post);
+  i++
+});
+
+
+
+clickTools = function(rowId, post) {
+  // provides upvote, downvote, edit, and delete buttons and reorders comments if needed based on up/down vote
+  var qualityUpButton = document.getElementById('upBtn' + i);
+  var qualityDownButton = document.getElementById('dwnBtn' + i);
+  var trashButton = document.getElementById('trash' + i);
+  var editButton = document.getElementById('edit' + i);
+  var row = document.getElementById(rowId)
+  var qualityUpButtons = document.getElementsByClassName('quality-up');
+  var buttonsCount = qualityUpButtons.length;
+  qualityUpButton.addEventListener('click', function() {
+    trQual = parseInt(row.getAttribute('qual')) + 1
```

Comment:
What is `trQual`? Could you name it differently? Also, was it ever declared with `var`?